video/filter/vf_gpu_vulkan.c

```diff
@@ -79,7 +79,7 @@ static struct offscreen_ctx *vk_offscreen_ctx_create(struct mpv_global *global,
         goto error;
 
     struct vulkan_opts *vk_opts = mp_get_config_group(NULL, global, &vulkan_conf);
-    vk->vulkan = mppl_create_vulkan(vk_opts, vk->vkinst, vk->pllog, VK_NULL_HANDLE);
+    vk->vulkan = mppl_create_vulkan(ractx->swapchain, vk->vkinst, vk->pllog, VK_NULL_HANDLE);
```

Comment:
We likely don't have swapchain here. Have you tested it? Also above line is dead code now.

Reply:
I can't test vf_gpu with vulkan atm so I don't wanna make changes to it blindly, for now I just added allow_software to mppl_create_vulkan